src/quattor-client/pom.xml

```diff
@@ -0,0 +1,123 @@
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.quattor.client</groupId>
+  <artifactId>quattor-client</artifactId>
+
+  <packaging>rpm</packaging>
+  <version>14.10.0-rc3-SNAPSHOT</version>
+  <name>Quattor meta RPM for bootstrapping client systems.</name>
+
+  <parent>
+    <groupId>org.quattor.maven</groupId>
+    <artifactId>build-profile</artifactId>
+    <version>1.32</version>
+  </parent>
+
+  <licenses>
+    <license>
+      <name>Apache 2 License</name>
+      <url>http://www.opensource.org/licenses/apache2.0</url>
+      <comments>Copyright (c) Responsible Organization</comments>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>James Adams</name>
+      <email>james.adams@stfc.ac.uk</email>
+    </developer>
+  </developers>
+
+  <build>
+    <plugins>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>tarballs</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>pod2man</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>filter-sources</id>
+            <phase>process-sources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration combine.self="override">
+              <outputDirectory>${project.build.directory}</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src</directory>
+                  <includes>
+                    <include>*.repo</include>
+                  </includes>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>rpm-maven-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <summary>Quattor client meta-package</summary>
+          <name>${project.artifactId}</name>
+          <url>https://github.com/quattor/release</url>
+          <requires>
+            <require>ncm-spma</require>
+            <require>ncm-cdispd</require>
+            <require>ccm</require>
+            <require>cdp-listend</require>
+            <require>ncm-query</require>
+            <require>ncm-ncd</require>
```

Comment:
Ok I've confirmed that `ncm-spma` is enough, it pulls in `perl-CAF`, which in turn pulls in `perl-AppConfig`.
I'll get rid of these.